client/blocks/inline-help/inline-help-search-results.jsx

```diff
@@ -140,16 +114,14 @@ function HelpSearchResults( {
 								if ( ! external ) {
 									event.preventDefault();
 								}
-								onLinkClickHandler( event, result );
+								onLinkClickHandler( event, result, type );
 							} }
 							{ ...( external && {
 								target: '_blank',
 								rel: 'noreferrer',
 							} ) }
 						>
-							{ support_type === SUPPORT_TYPE_ADMIN_SECTION && (
-								<Gridicon icon={ icon } size={ 18 } />
-							) }
+							{ type === SUPPORT_TYPE_ADMIN_SECTION && <Gridicon icon={ icon } size={ 18 } /> }
```

Comment:
This condition could also be: `icon && <Gridicon icon={ icon }`. I.e., if the result has an `.icon` field, render the icon. That's better than checking the `type`.

Reply:
I need to rebase this branch, this change should already be in `trunk`.